ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/MLChatAgentRunner.java

```diff
@@ -652,4 +657,34 @@ private void runReAct(
         client.execute(MLPredictionTaskAction.INSTANCE, request, firstListener);
     }
 
+    private String outputToOutputString(Object output) throws PrivilegedActionException {
```

Comment:
are these utility methods to handle format transformation? If so, is it possible to move them to a utility class under utils that can be reused somewhere else?

Reply:
Already do it.